docs/book/src/developer/tilt.md

```diff
@@ -257,7 +280,28 @@ create a cluster. There are [example worker cluster
 configs](https://github.com/kubernetes-sigs/cluster-api/tree/main/test/infrastructure/docker/examples) available.
 These can be customized for your specific needs.
 
-<aside class="note">
+### Deploying a workload cluster
+
+After your kind management cluster is up and running with Tilt, you can deploy a workload clusters in the Tilt web UI based off of YAML templates from specified directories. By default, templates are read from `./test/infrastructure/docker/templates`.
+
+These deployment resources are found in the Tilt web UI under the label grouping `<provider>-flavors` and `<provider>-clusterclasses` for each specified provider, i.e. `docker-flavors` and `docker-clusterclasses`. Note that deploying a workload cluster from Tilt UI is also termed as flavor cluster deployment.
```

Comment:
```suggestion
These deployment resources are found in the Tilt web UI under the label grouping `<provider>-flavors` and `<provider>-clusterclasses` for each specified provider, i.e. `docker-flavors` and `docker-clusterclasses`. Note that deploying a workload cluster from Tilt UI is also termed as cluster flavor deployment.
```
? not sure

To be honest I'm hearing the term "flavor cluster" for the first time. Maybe just drop the last sentence, for me it sounds like we are establishing a new term which isn't really necessary here - and at least for me it is confusing